hxrsnd/plans/scans.py

```diff
@@ -85,7 +71,7 @@ def linear_scan(motor, start, stop, num, use_diag=True, return_to_start=True,
     start = motor.position
     
     # Define the inner scan
-    @stage_decorator([motor])
+    # @stage_decorator([motor])
```

Comment:
The fact you needed to comment this out is very strange because EpicsMotor doesn't have a stage method.

Reply:
I was also confused by this. I want to do more testing whenever I get the chance.